prefect_hightouch/exceptions.py

```diff
@@ -0,0 +1,51 @@
+"""
+This is a module containing exceptions used within prefect-hightouch.
+"""
+
+from prefect_hightouch.api_client.models import SyncStatus
+
+
+class HightouchSyncRunError(RuntimeError):
```

Comment:
Big fan of this exception structure!

Reply:
:) Thanks, inspired by your work in dbt Cloud, Nate separating into exceptions, and the Dan Bader's Python Tricks book!